__tests__/comment.spec.ts

```diff
@@ -37,10 +37,10 @@ describe('composeCommentsForUsers', () => {
         },
       ])
     ).toMatchInlineSnapshot(`
-        Array [
-          "This is a custom message for a rule",
-        ]
-      `);
+      Object {
+        "/Users/cyamonide/GitHub/use-herald-action/some/rule.json": "This is a custom message for a rule",
```

Comment:
this should be changedto have a dynamic path instead, or it will fail tests on github.

Reply:
whoops good catch - I did this somewhere else but missed it here